app/models/profile_field.rb

```diff
@@ -1,13 +1,11 @@
 class ProfileField < ApplicationRecord
   # Key names follow the Rails form helpers
-  INPUT_TYPES = {
+  enum input_type: {
```

Comment:
If you want to make these more human-readable in Blazer you can add this column name and enum descriptions [here to our Blazer.yml file](https://github.com/forem/forem/blob/master/config/blazer.yml#L33)

Reply:
Oh yes, let's please do this. 

Rails enums can also be backed by real [Postgres enums](https://www.postgresql.org/docs/current/datatype-enum.html#id-1.5.7.15.7) but we don't use that anywhere else in the app. The one drawback is that you need to add a migration for new enum values (`ALTER TYPE name ADD VALUE ...`), but otherwise I prefer that over meaningless integer columns.

Reply:
Done ✅

I do have a very noob question on this after looking through the documentation though. I found it odd that we don't have a table name attached to to the smart column. What if there's a column name in a different table whose enums are different?